bigbluebutton-html5/imports/ui/components/lock-viewers/notify/component.jsx

```diff
@@ -20,6 +20,10 @@ const intlMessages = defineMessages({
     id: 'app.userList.userOptions.disablePubChat',
     description: 'label to disable private chat notification',
   },
+  disableNote: {
```

Comment:
Seems unused

Reply:
Are you sure? Looks right to me. Please ask who did the lock notifier if this is enough